R/manual_layer_node.R

```diff
@@ -367,7 +367,35 @@ setMethod("compute_sequentially", signature(object = "Node"), function(object) {
   object$compute_sequentially()
 })
 
-# ----------------------------------------------------------------
+##' Get arguments for a delayed function, as a list.
+##'
+##' @family {manual-layer functions}
+##' @export
+args <- function(object, ...) 0
```

Comment:
That 0 looks like a typo here.  Can you make it `NULL` or just (maybe more idiomatic) `{}`  ?

`args()` is an existing base R function so maybe the naming here is too aggressive as it shadows a base function? But it looks like your use of a generic added here prevents harm.  If that does in fact work, great!   (May be prudent to have a unit test ir or two 'calibrated' on base R's `args()` in the package to ensure it still works after package load? Or maybe that is overkill.

Reply:
It is indeed a problem as I suspected but I once again fell into the rabbit hole of looking at this before other commit came in.   As it stands right now, R is pretty clear about this:

```r
> library(tiledbcloud)

Attaching package: ‘tiledbcloud’

The following object is masked from ‘package:base’:

    args

> 
```